<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Collatz Sequence</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Limit (Max Number):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="limitSpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
      <property name="value">
       <number>100000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelThreads">
      <property name="text">
       <string>Threads:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="threadSlider">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>8</number>
      </property>
      <property name="value">
       <number>4</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="resultText">
      <property name="text">
       <string>Results will appear here...</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <connections/>
</ui>
